dolphinscheduler-common/src/main/java/org/apache/dolphinscheduler/common/utils/JSONUtils.java

```diff
@@ -196,7 +196,10 @@ public static <T> List<T> toList(String json, Class<T> clazz) {
      * @return true if valid
      */
     public static boolean checkJsonValid(String json) {
+        return checkJsonValid(json, true);
+    }
 
+    public static boolean checkJsonValid(String json, Boolean logFlag) {
```

Comment:
Why not log default if parse wrong? I think it's not need to add this log flag.

Reply:
Looks like use this method to check a string is a json or not

Reply:
@caishunfeng The original method has not changed, only enhancements have been made to the original method. For the adaptation of the generated file extension, no log output is required. Adding this log parsing is of little significance and may cause some confusion to users.